### DISCUSSION TITLE
Is it possible to use this awesome project with only a 32u2?

### DISCUSSION THREAD

**Discussion by developer-ken** [NicoHood/HoodLoader2#101] [Q&A]
I need to design a really compact board, and it will be nice if it can be programmed with Arduino IDE.
I found this awesome project, but in the wiki, a 32u2 is always combined with another chip. Will this project work just fine when a 32u2 is used alone?

**Comment by NicoHood** [comment]
Yes it will. But you could just use a normal CDC Bootloader from LUFA. Hoodloader2 gives you the serial bridge in bootloader mode. If you dont need it, use the CDC Bootloader. If you do, use this. All of them work.
